pkg/controller/installation/products/fuse/reconciler.go

```diff
@@ -191,28 +192,41 @@ func (r *Reconciler) reconcileImageVersion(ctx context.Context, install *v1alpha
 	return v1alpha1.PhaseFailed, errors.New("Could not find trigger for postgres:9.5 in deploymentconfig")
 }
 
-func (r *Reconciler) reconcileAdminPerms(ctx context.Context, client pkgclient.Client) (v1alpha1.StatusPhase, error) {
-	r.logger.Infof("Reconciling permissions for %s group on %s namespace", adminGroupName, r.Config.GetNamespace())
+// Ensures all users in rhmi-developers group have view Fuse permissions
+func (r *Reconciler) reconcileViewFusePerms(ctx context.Context, client pkgclient.Client) (v1alpha1.StatusPhase, error) {
+	r.logger.Infof("Reconciling view Fuse permissions for %s group on %s namespace", developersGroupName, r.Config.GetNamespace())
 
-	roleName := adminGroupName + "-view-fuse"
-	adminViewFuseRole := &rbacv1.ClusterRole{
+	roleName := developersGroupName + "-view-fuse"
```

Comment:
There is an existing cluster ``` view ``` role. Why not create a rolebinding in the namespace to that role?

Reply:
Didn't know that role existed. I'll update this so that it points to that cluster role instead. We can then remove this additional cluster role. Users will be able to view the Fuse CR with this role, is that ok?